src/manage_nsfs/nc_lifecycle.js

```diff
@@ -210,23 +214,31 @@ async function process_rules(config_fs, bucket_json, object_sdk, should_notify)
     }
 }
 
-async function send_lifecycle_notifications(delete_res, bucket_json, object_sdk) {
+function create_notification_delete_object(delete_res, delete_obj_info) {
```

Comment:
1. please move to notifications util section
2. please add jsdoc

Reply:
do we need to pass the version id of the delete marker in the notification as well? what about deleted_delete_marker? will we need this info?

Reply:
right its not part of the object_info because we get the object info of the object to delete. I can pass it from the res. there are many nuances I am adding tests now